pkg/network/networker.go

```diff
@@ -1277,14 +1275,24 @@ func (n *networker) ZOSAddresses(ctx context.Context) <-chan pkg.NetlinkAddresse
 		return result
 	}
 
+	done := make(chan struct{})
+	updateChan := make(chan netlink.AddrUpdate)
+	err := netlink.AddrSubscribe(updateChan, done)
```

Comment:
I think you can directly use `ctx.Done()`  here directly instead of creating a done channel that only closes when ctx is cancelled